### PR TITLE
Colin/apply safe version ranges

### PR DESCRIPTION
**Pull request by colin-higgins**
Apply version ranges to each relevant integration.
Cleanup unused namespaces.
Update devenv.bat.

### CODE REVIEW

File: src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNetIntegration.cs [hunk 1]
```diff
@@ -1,9 +1,7 @@
 #if !NETSTANDARD2_0
```
Comment (lucaspimentel) [inline src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNetIntegration.cs]: Let's delete this file. We're not using `AspNetIntegration`.

File: src/Datadog.Trace.ClrProfiler.Managed/Integrations/StackExchange.Redis/ConnectionMultiplexer.cs [hunk 2]
```diff
@@ -24,12 +24,16 @@ public static class ConnectionMultiplexer
             Integration = IntegrationName,
             CallerAssembly = "StackExchange.Redis",
             TargetAssembly = "StackExchange.Redis",
-            TargetType = "StackExchange.Redis.ConnectionMultiplexer")]
+            TargetType = "StackExchange.Redis.ConnectionMultiplexer",
+            TargetMinimumVersion = Major1,
+            TargetMaximumVersion = Major1)]
```
Comment (lucaspimentel) [inline src/Datadog.Trace.ClrProfiler.Managed/Integrations/StackExchange.Redis/ConnectionMultiplexer.cs]: Maximum version should be "2". `StackExchange.Redis` is up to `2.0.601` and afaik the integration works with 2.x.

File: src/Datadog.Trace.ClrProfiler.Managed/Integrations/ServiceStackRedisIntegration.cs [hunk 3]
```diff
@@ -25,7 +25,9 @@ public static class ServiceStackRedisIntegration
         [InterceptMethod(
             CallerAssembly = "ServiceStack.Redis",
             TargetAssembly = "ServiceStack.Redis",
-            TargetType = "ServiceStack.Redis.RedisNativeClient")]
+            TargetType = "ServiceStack.Redis.RedisNativeClient",
+            TargetMinimumVersion = Major4,
+            TargetMaximumVersion = Major4)]
```
Comment (lucaspimentel) [inline src/Datadog.Trace.ClrProfiler.Managed/Integrations/ServiceStackRedisIntegration.cs]: Maximum version should be "5". `ServiceStack.Redis` is up to `5.5.0` and afaik the integration works with 5.x.
https://www.nuget.org/packages/ServiceStack.Redis/

File: src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNetCoreMvc2Integration.cs [hunk 4]
```diff
@@ -263,7 +268,9 @@ public AspNetCoreMvc2Integration(object actionDescriptor, object httpContext)
         [InterceptMethod(
             CallerAssembly = "Microsoft.AspNetCore.Mvc.Core",
             TargetAssembly = "Microsoft.AspNetCore.Mvc.Core",
-            TargetType = ResourceInvoker)]
+            TargetType = ResourceInvoker,
```
Comment (zacharycmontoya) [inline src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNetCoreMvc2Integration.cs]: I know this isn't new, but is there a reason this isn't referenced by full name, like `Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker`?

**Comment by zacharycmontoya** [comment]
This PR looks good to me but I'd prefer @lucaspimentel approve since I have less context on the supported version numbers.

File: src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNetCoreMvc2Integration.cs (hunk 4 above)
Comment (colin-higgins) [reply src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNetCoreMvc2Integration.cs]: Ah yeah, it is the fully qualified name, it's in the const higher up in the file.

**Pull request merged**